tests/helpers/perf_run_helper.rb

```diff
@@ -524,6 +524,9 @@ def copy_scale_results(scenario)
     FileUtils.copy_file "#{@archive_root}/start_epoch", "#{scale_result_dir}/start_epoch"
     FileUtils.copy_file "#{@archive_root}/end_epoch", "#{scale_result_dir}/end_epoch"
 
+    # copy the pe_xl build directory to the results if it exists
+    FileUtils.cp_r "build", "#{scale_results_parent_dir}/build" if Dir.exist?("build")
```

Comment:
build dir should probably get a module wide variable to hold it's name... like PERF_RESULTS_DIR
Small thing...